src/main/java/org/prgrms/cream/domain/deal/service/SellingService.java

```diff
@@ -0,0 +1,60 @@
+package org.prgrms.cream.domain.deal.service;
+
+import java.time.format.DateTimeFormatter;
+import org.prgrms.cream.domain.deal.domain.SellingBid;
+import org.prgrms.cream.domain.deal.dto.BidRequest;
+import org.prgrms.cream.domain.deal.dto.BidResponse;
+import org.prgrms.cream.domain.deal.repository.SellingRepository;
+import org.prgrms.cream.domain.product.domain.ProductOption;
+import org.prgrms.cream.domain.product.service.ProductService;
+import org.prgrms.cream.domain.user.exception.InvalidArgumentException;
+import org.prgrms.cream.domain.user.service.UserService;
+import org.prgrms.cream.global.error.ErrorCode;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class SellingService {
+
+	private final SellingRepository sellingRepository;
+	private final ProductService productService;
+	private final UserService userService;
+
+	public SellingService(
+		SellingRepository sellingRepository,
+		ProductService productService,
+		UserService userService
+	) {
+		this.sellingRepository = sellingRepository;
+		this.productService = productService;
+		this.userService = userService;
+	}
+
+	@Transactional
+	public BidResponse registerBuyingBid(Long id, String size, BidRequest bidRequest) {
+		ProductOption productOption = productService.findProductOptionByProductIdAndSize(id, size);
+
+		if (bidRequest.price() > productOption.getSellHighestPrice())
+			productOption.updateSellHighestPrice(bidRequest.price());
+		else
+			throw new InvalidArgumentException(ErrorCode.INVALID_INPUT);
```

Comment:
조건절의 로직이 한줄이어도 중괄호로 구분지어주는 것이 좋다고 생각되는데 수형님은 어떻게 생각하시나용
```suggestion
		if (bidRequest.price() > productOption.getSellHighestPrice()) {
			productOption.updateSellHighestPrice(bidRequest.price());
		} else {
			throw new InvalidArgumentException(ErrorCode.INVALID_INPUT);
		}
```

Reply:
확실히 중괄호를 작성해주는게 훨씬 나아보입니다!! 이 부분 바로 수정하겠습니다!!

Reply:
5dcb587 : 수정했습니다!!